parser/parser.y

```diff
@@ -1569,6 +1579,42 @@ AlterTableStmt:
 		}
 	}
 
+ResourceGroupOptionList:
+	DirectResourceGroupOption
+	{
+		$$ = []*ast.ResourceGroupOption{$1.(*ast.ResourceGroupOption)}
+	}
+|	ResourceGroupOptionList DirectResourceGroupOption
+	{
+		$$ = append($1.([]*ast.ResourceGroupOption), $2.(*ast.ResourceGroupOption))
+	}
+|	ResourceGroupOptionList ',' DirectResourceGroupOption
+	{
+		$$ = append($1.([]*ast.ResourceGroupOption), $3.(*ast.ResourceGroupOption))
+	}
+
+DirectResourceGroupOption:
+	"RRU_PER_SEC" EqOpt stringLit
+	{
+		$$ = &ast.ResourceGroupOption{Tp: ast.ResourceRRURate, StrValue: $3}
+	}
+|	"WRU_PER_SEC" EqOpt stringLit
+	{
+		$$ = &ast.ResourceGroupOption{Tp: ast.ResourceWRURate, StrValue: $3}
+	}
+|	"CPU" EqOpt stringLit
```

Comment:
So, we can say "create resource group x rru_per_sec='2000', cpu = '8'". That is specify rru_per_sec and cpu at the same time?  The semantic is a little weird. I prefer to make 2 groups, either rru&wru or cpu/iobandwidth.

Reply:
Here only parser part. we can limit the behavior in DDL handle logic. `ru` and `cpu/bandwith` cannot be specified at the same time, report the error to user.